mne/cov.py

```diff
@@ -1713,6 +1713,8 @@ def _estimate_rank_meeg_signals(data, info, scalings, tol=1e-4,
         thresholded to determine the rank are also returned.
     """
     picks_list = _picks_by_type(info)
+    if scalings is None:
+        scalings = dict(mag=1e15, grad=1e13, eeg=1e6)
```

Comment:
Yeah we should DRY it away. I wonder if we can just use one set of scalings everywhere. `make_ad_hoc_cov` might need its own set of values, if that's the sticking point.